### DISCUSSION TITLE
Find out token for ex team members as admin

### DISCUSSION THREAD

**Discussion by falschermarcus** [reportportal/reportportal#1756] [Q&A] [answered]
Hej Team,

I was curious if there is a way as report portal admin to see or access the token of a former member?

We currently have the case that a member left the company. In the member overview I can only see the member name, their assigned projects and roles and additional information such as email and last login date.

In order to change the rpConfig file where the token is provided I need to make sure the token belongs to the ex team member before we make any changes.

Is there such a function or option? We have no way of accessing or logging in with that user anymore. 

Best regards and thanks in advance!

**Comment by MaryiaRymarchyk** [comment, marked as answer]
Hey @falschermarcus,
Unfortunately you can't get token from another user via UI, it can be found only in database.
Let me know if you need more help here.

**Comment by MaryiaRymarchyk** [comment]
@falschermarcus long time no hear from you. 
Seems that there are no remain actions from our side, so we'll close the issue for now.
If you have questions -  just get in touch.
